security/rbac/scripts/enable-rbac-ksql-server.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
```

Comment:
should this script (or other scripts, for that matter) run with `-e` flag, to exit if something goes wrong?

Reply:
@stan-confluent thanks, I will consider this in a separate Jira DEVX-922 for all services, not just KSQL